core/types/transaction_meta.go

```diff
@@ -121,6 +123,14 @@ func TxMetaDecode(input []byte) (*TransactionMeta, error) {
 		meta.QueueIndex = &queueIndex
 	}
 
+	raw, err := common.ReadVarBytes(b, 0, 130000, "RawTransaction")
```

Comment:
Should we get rid of the concept of max read size? This is `0.13mb` which should be enough but I don't know the consensus tx size limit off of the top of my head